M2/Macaulay2/packages/ToricHigherDirectImages.m2

```diff
@@ -34,6 +34,7 @@ newPackage(
 export {
     -- types
     -- methods
+    "frobeniusPushforward",
     "frobeniusDirectImage",
```

Comment:
A cleaner way to declare a synonym would be to write 
```
"frobeniusDirectImage" => "frobeniusPushforward"
```
(on line 38) and then get rid of anythin that mentions `frobeniusDirectImage` in the rest of code and documentation.